### PR TITLE
feat: add toast

### LINKED ISSUE #21: add toast

### PR DESCRIPTION
**Pull request by Bhakti199**
Here I added toast. #21

### CODE REVIEW

File: src/Components/CartCards/Card.css [hunk 1]
```diff
@@ -7,3 +7,11 @@
 .icon-size {
   font-size: 1.5rem;
 }
+@media screen and (max-width: 500px) {
```
Comment (Raghav888) [inline src/Components/CartCards/Card.css:10]: Use em here

File: src/Components/CartCards/ProductCardForCart.jsx [hunk 2]
```diff
@@ -77,6 +79,7 @@ export const ProductCardForCart = () => {
                       payload: item.cartItem,
                     });
                     dispatch({ type: "REMOVE_FROM_CART", payload: item });
+                    toast("added to wishlist", { icon: "✔️" });
```
Comment (Raghav888) [inline src/Components/CartCards/ProductCardForCart.jsx:82]: Like that you used toast and icons for it

**Review by Raghav888 — APPROVED** [review]

File: src/Components/CartCards/ProductCardForCart.jsx [hunk 3]
```diff
@@ -63,9 +64,10 @@ export const ProductCardForCart = () => {
               <div className="horizontal-card-btn flex-col-center">
                 <button
                   className="primary-btn-md-two"
-                  onClick={() =>
-                    dispatch({ type: "REMOVE_FROM_CART", payload: item })
-                  }
+                  onClick={() => {
+                    dispatch({ type: "REMOVE_FROM_CART", payload: item }),
+                      toast("removed from cart", { icon: "❌" });
```
Comment (JantuDeb) [inline src/Components/CartCards/ProductCardForCart.jsx:69]: I think you have added a `,` instead of `;`

**Review by rkapoor10 — APPROVED** [review]
- Liked the implementation of React toast
- Nice implementation of Hamburger menu option with check conditions
- I learned the use of icons through objects 
- not sure why yarn.lock is showing in files changed

Overall the code is well structured and good to merge

File: src/Components/ProductCard/ProductCard.jsx [hunk 4]
```diff
@@ -17,18 +18,20 @@ export const ProductCard = () => {
...
-                    onClick={() =>
+                    onClick={() => {
                       dispatch({
                         type: "REMOVE_FROM_WISHLIST",
                         payload: item,
-                      })
-                    }
+                      }),
+                        toast("removed from wishlist", { icon: "❌" });
+                    }}
```
Comment (JantuDeb) [inline src/Components/ProductCard/ProductCard.jsx:27]: If you are calling two or more function in` onClick` then use a `clickHandler` function.

**Review by JantuDeb — APPROVED** [review]
It's looks really awesome, you have added  toast for actions. 
Added few suggestion otherwise the code looks fine

**Pull request merged**